src/DataTransformer/BooleanTypeToBooleanTransformer.php

```diff
@@ -37,9 +37,9 @@ public function transform($value): ?int
     }
 
     /**
-     * @param mixed $value
+     * @phpstan-param mixed $value
```

Comment:
why do we need this if you already added the typehint?

Reply:
Because phpstan consider from
```
@phpstan-implements DataTransformerInterface<bool, int>
```
than the phpdoc is
```
@param bool|null $value
```

Reply:
So, the interface expect mixed as typehint then?